microsoft-edge/progressive-web-apps-chromium/ux.md

```diff
@@ -0,0 +1,108 @@
+---
+description: This guide gives you an overview of the user experience of PWAs on Microsoft Edge and Windows.
+title: The user experience of PWAs
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 09/15/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: progressive web apps, PWA, Edge, Windows, desktop, installation, integration, microsoft store, ux
+---
+# The user experience of PWAs  
+
+On Windows, PWAs are first-class app citizens and any device running Microsoft Edge gets full access to the technologies and characteristics of Progressive Web Apps.  
+
+## Installing a PWA  
+
+When Microsoft Edge determines that a website is an installable PWA, the **App available** button is displayed in the address bar.  
+
+Select the **App available** button to install the PWA.  
+
+:::image type="complex" source="./media/edge-app-install-flyout.png" alt-text="The installation prompt in Microsoft Edge" lightbox="./media/edge-app-install-flyout.png":::
+   The installation prompt in Microsoft Edge  
+:::image-end:::  
+
+Select **Install** to complete the installation and run the PWA in Windows.  
+
+Many PWAs are also present on the Microsoft Store and can be installed directly from there without even opening Microsoft Edge.  
+
+To install a PWA from the Microsoft Store, select **Get** on the app page.  
+
+:::image type="complex" source="./media/install-twitter-microsoft-store.png" alt-text="Installing an app from the Microsoft Store" lightbox="./media/install-twitter-microsoft-store.png":::
```

Comment:
I’d suggest putting the box around the button rather than in it. It’s difficult to distinguish the red box from the blue button background. You might even consider an oval instead, to draw more attention in a sea of rectangles.